esp/templates/errors/program/wronggrade.html

```diff
@@ -14,7 +14,7 @@
 <br />
 <em>Sorry, you do not appear to match the required grade range for this program. Please browse through
 some of our other <a href="/learn/index.html">programs</a>, though!</em><br />
-If your grade, <strong>{{request.user.getGrade}}</strong>, is incorrect, please <a href="/myesp/profile" title="ESP Profile Editor">edit your profile</a> or <a href="mailto:{{ DEFAULT_EMAIL_ADDRESSES.default }}">contact an administrator</a> to change it.
+If your graduation year, <strong>{{yog}}</strong>, is incorrect, please <a href="/myesp/profile" title="ESP Profile Editor">edit your profile</a> or <a href="mailto:{{ DEFAULT_EMAIL_ADDRESSES.default }}">contact an administrator</a> to change it.
```

Comment:
I'm not sure we want to make it more obvious that you can change your grade and get into a program that you don't have access to? Shrug.